analytic_hr_department_restriction/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2025 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Analytic Hr department Restriction",
```

Comment:
Please ellaborate more the description of the modules, not to put the technical name in capitals (and in fact, `Hr` is incorrect, in any case it should be `HR`):

```suggestion
    "name": "Analytic distributions restriction per HR department",
```